tools/triage/add_triage_signature.py

```diff
@@ -2983,6 +2983,68 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("see MGMT-15243")
 
 
+class DuplicateVIP(ErrorSignature):
+    """
+    Looks for nodes holding the same VIP
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. VIP found in multiple nodes",
+            function_impact_label="vip_duplicate",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        cluster = get_metadata_json(url)["cluster"]
+
+        # SNO doesn't need balancing
+        if cluster.get("high_availability_mode") == "None":
+            return
+
+        # VIPs are not relevant with load balancer
+        if cluster.get("user_managed_networking") == True:
```

Comment:
```suggestion
        if cluster.get("user_managed_networking"):
```

Reply:
If `user_managed_networking` has any value other than `True` (e.g., `5`) then I prefer this `if` to not have its body executed, so your diff suggestion isn't technically equivalent.

The concept of values having "truthiness" is in my opinion a very unfortunate unnecessary complication that sadly most programming languages have which I prefer to not rely on.

Reply:
I agree, but linter disagrees :)

Reply:
We could ignore linting though